cypress/integration/crud.spec.js

```diff
@@ -1,36 +1,16 @@
-describe('Notes', () => {
-  beforeEach(() => {
-    cy.visit('http://notes-serverless-app.com')
-
-    cy.get('.navbar-nav a:contains(Login)').click()
-
-    cy.get('#email').type(Cypress.env('user'))
-    cy.get('#password').type(Cypress.env('password'))
-    cy.get('button[type="submit"]').click()
-  })
+/// <reference types="Cypress" />
 
-  it('creates a note', () => {
-    cy.contains('Create a new note').click()
-
-    cy.get('#content').type('My note')
-    cy.contains('Create').click()
-
-    cy.get('.list-group').should('contain', 'My note')
-  })
-
-  it('edits a note', () => {
-    cy.get('.list-group').contains('My note').click()
-    cy.get('#content').type(' updated')
-    cy.contains('Save').click()
-
-    cy.get('.list-group').should('contain', 'My note updated')
-    cy.get('.list-group:contains(My note updated)').should('be.visible')
-  })
-
-  it('deletes a note', () => {
-    cy.get('.list-group').contains('My note updated').click()
-    cy.contains('Delete').click()
-
-    cy.get('.list-group:contains(My note updated)').should('not.exist')
+describe('Notes', () => {
+  beforeEach(() => cy.login(Cypress.env('user'), Cypress.env('password')))
```

Comment:
What about just `cy.login()`, defaulting the user and password directly on the custom command?

Reply:
@wlsf82 Nice! Let me try and I'll update it.

Reply:
Done!